samples/apps/copilot-chat-app/README.md

```diff
@@ -0,0 +1,77 @@
+# Copilot Chat Sample Application
+>! IMPORTANT  This learning sample is for educational purposes only and should not be used in any production use case.  It is intended to highlight concepts of Semantic Kernel and not any architectural / Security design practices to be used.
+
+## About the Copilot 
+The Copilot Chat sample allows you to build your own integrated large language model chatbot.  This is an enriched intelligence app, with multiple dynamic components including command messages, user intent, and memories.  The chat prompt and response will evolve as the conversation between the user and the application proceeds.  This chat experience is a chat skill containing multiple functions that work together to construct the final prompt for each exchange.
+
+
+![UI Sample](images/UI-Sample.png)
+
+## Dependencies:
+
+Before following these instructions, please ensure your development environment and these components are functional:
```

Comment:
[nit] I would like to remove assumptions in these statements. For example,
```
1.	[Visual Studio Code](https://code.visualstudio.com/Download)
2.	[Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
3.	[.NET 6.0](https://dotnet.microsoft.com/en-us/download/dotnet/6.0)
4.	[Node.js](https://nodejs.org/en/download)
5.	[Yarn](https://classic.yarnpkg.com/lang/en/docs/install)
```

Reply:
Done.